### ISSUE TITLE
When activated, the view: ShowScm command opens an editor, but orca does not send any information messages about the editor

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#93574]
Issue Type: <b>Bug</b>

1. Launch orca.
2. launch vscode using a folder belonging to some git repository.
3. Make sure that the repository has at least one file changed.
4. Press ctrl + shift + g.

An editor is open but there is no description about the purpose of that editor. I understood that we could type a message to a commit, but I wasn't sure if that was it.
@isidorn  @sana-ajani When time permits it would be possible to check to make sure I am not doing / talking nonsense!

Thanks.

VS Code version: Code - Insiders 1.44.0-insider (6cf3dd97465b2f4f666b66758991d720e003b0f4, 2020-03-27T05:34:30.255Z)
OS version: Linux x64 5.5.11-arch1-1

**Comment by isidorn** [comment]
The scm input box should have an aria label. I can look into this.
fyi @joaomoreno

**Comment by isidorn** [comment]
@jvesouza Thanks for filling this issue. We have pushed a fix, the next VS Code insiders should include it. It would be great if you can try it out and let us know if the issue got fixed on your side. Thanks!

**Comment by jvesouza** [comment]
@isidorn IMHO was excellent.
Thanks.

**Comment by isidorn** [comment]
Thanks for letting us know, adding verified label.
